lib/veri_helpers.ml

```diff
@@ -0,0 +1,47 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std
+
+module Dis = Disasm_expert.Basic
+
+(** [name_of_insns insns] - returns a name of last instruction in [insns]  *)
+let name_of_insns insns = 
+  match List.hd (List.rev insns) with
+  | None -> Or_error.error_string "instruction hasn't name"
+  | Some (_, insn) -> Ok (Insn.(name (of_basic insn)))
+
+let insns_of_mem dis mem = 
```

Comment:
what about instruction prefix, that goes as stand alone instruction when we are using llvm?